react/src/components/dashboard/dashboard.scss

```diff
@@ -1,3 +1,63 @@
-.dashboard {
+.full-height {
+  height: 100%;
+}
 
+.text-align-center {
+  text-align: center;
+}
+
+.display-none {
+  display: none !important;
+}
+
+.margin-top-30 {
+  margin-top: 30px;
+}
+
+ jumblr.js
+.jumblr-header {
+  font-size: 24px;
+  text-align: center;
+}
+
+// loginModal.js
+.color-white {
+  color: #fff;
+}
+
+//navbar.js
+.padding-top-16 {
+  padding-top: 16px !important;
```

Comment:
I added the '!important' part because there already was 22px top padding for the anchor tag from a different rule, and when I moved this style from inline to external, the 22px top padding rule took preference.

I now created a more specific rule so that the '!important' part is no longer needed.